app/controllers/admin/talentlms_courses_controller.rb

```diff
@@ -46,6 +46,8 @@ def course_params
         :courseid,
         :months_to_expiration,
         :default,
+        :start_date,
```

Comment:
I can't comment above, but the courses should probably be sorted in some way that places the active courses first in the table in date order.

Reply:
I'll plan to add some sorting to the course to address this